packages/vc-verification/src/utils/formatters.ts

```diff
@@ -6,6 +6,7 @@ import type { RoleCredentialSubject } from '@energyweb/credential-governance';
 import type { VerifiableCredential } from '@ew-did-registry/credentials-interface';
 import { Chain } from '@ew-did-registry/did';
 import { utils } from 'ethers';
+import _ from 'lodash';
```

Comment:
Can the lodash import be optimized a bit? https://stackoverflow.com/questions/35250500/correct-way-to-import-lodash

Reply:
Ah, yeah, I missed that

Reply:
Ok, no worries. I see you requested review but import is still unchanged. Is there a change incoming still here?

Reply:
Sorry. Changes were not pushed for some reason. Should be ok now